workflows/diagnostics/fv3net/diagnostics/prognostic_run/load_run_data.py

```diff
@@ -243,6 +243,12 @@ def data_2d(self) -> xr.Dataset:
     def data_3d(self) -> xr.Dataset:
         return _load_3d(self.url, self.catalog)
 
+    @property
+    def artifacts(self) -> List[str]:
+        url = self.url
+        fs = vcm.get_fs(url)
```

Comment:
Is this subject to the caching issue you identified earlier? We could override `ls` on the `fs` returned by `get_fs` to always invalidate the cache